app/Rakefile

```
@@ -34,4 +34,23 @@ namespace :app do |args|
     update_timestamps!(:users, [admin])
     data_adapter.upsert(:users, resources: [admin])
   end
-end
+
+  desc "checks if Admin user exists"
+  task :does_admin_exist? do
```

Nit: I think it's slightly more idiomatic to adjust the method name this way

```suggestion
  task :admin_exists? do
```

---

That makes sense to me, I committed your change and updated the task name in the[ PDS module PR](https://github.com/puppetlabs/puppetlabs-puppet_data_service/pull/7/commits/9e0f38a168ba8d31de118effc34c5f06f156b13d)